src/OpenLoco/src/Ui/Widgets/ScrollViewWidget.cpp

```diff
@@ -11,229 +11,207 @@ namespace OpenLoco::Ui::Widgets
     static void drawHScroll(Gfx::DrawingContext& drawingCtx, const Widget& widget, const WidgetState& widgetState, const ScrollArea& scrollArea)
     {
         auto* window = widgetState.window;
+        const auto position = window->position() + widget.position();
+        const auto size = widget.size();
 
-        uint16_t left = window->x + widget.left + 1;
-        uint16_t top = window->y + widget.top + 1;
-        uint16_t right = window->x + widget.right - 1;
-        uint16_t bottom = window->y + widget.bottom - 1;
+        // Calculate adjusted dimensions
+        auto scrollPos = Ui::Point{ position.x + 1, position.y + size.height - 11 }; // top = bottom - 10
```

Comment:
We're using 10 +/- 1 a lot in this function. I think a constant, e.g. `kScrollbarSize = 10` would be useful.